src/pages/administration/teachers/index.vue

```diff
@@ -34,15 +34,15 @@
 			<template v-slot:datacolumn-createdAt="{ data }">
 				{{ dayjs(data).format("DD.MM.YYYY") }}
 			</template>
-			<template v-slot:datacolumn-consent="{ data }">
-				<span v-if="data && data.consentStatus === 'ok'">
+			<template v-slot:datacolumn-consentStatus="{ data }">
+				<span v-if="data === 'ok'">
```

Comment:
maybe rename data to status for better readability?
```suggestion
			<template v-slot:datacolumn-consentStatus="{ data: status }">
				<span v-if="status === 'ok'">
```